### ISSUE TITLE
Misplacement of arguments in non_saturating_d_loss and non_saturating_gen_loss

### ISSUE THREAD

**Issue by sebgao** [FoundationVision/LlamaGen#84]
Although `non_saturating_d_loss` and `non_saturating_gen_loss` are not default choices in this codebase, there seems a mistake of argument placement `label` and `input` in these two functions, which should be swapped (`input` and `label`).
https://github.com/FoundationVision/LlamaGen/blob/ce98ec41803a74a90ce68c40ababa9eaeffeb4ec/tokenizer/tokenizer_image/vq_loss.py#L29-L30

https://github.com/FoundationVision/LlamaGen/blob/ce98ec41803a74a90ce68c40ababa9eaeffeb4ec/tokenizer/tokenizer_image/vq_loss.py#L40

Also check https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html#torch.nn.BCEWithLogitsLoss.
